hs_app_netCDF/nc_functions/nc_meta.py

```diff
@@ -59,7 +59,7 @@ def get_nc_meta_dict(nc_file_name):
     return nc_meta_dict
 
 
-# Functions for dublin  core meta ###############################################################
+# Functions for dublin core meta
```

Comment:
Would be better to remove this comment here and if needed same comment can be included as part of the function docstring.